boards/arm/nrf5340_dk_nrf5340/doc/index.rst

```diff
@@ -257,4 +266,6 @@ References
 
 .. _IDAU:
    https://developer.arm.com/products/architecture/cpu-architecture/m-profile/docs/100690/latest/attribution-units-sau-and-idau
+.. _nRF53 DK website:
+   https://www.nordicsemi.com/Software-and-tools/Development-Kits/nRF5340-PDK
 .. _Nordic Semiconductor Documentation library: https://www.nordicsemi.com/DocLib
```

Comment:
DocLib is not longer being used.  We should link to Infocenter instead.

Reply:
See PR #20765